src/test/java/com/microsoft/azure/servicebus/jms/aad/SendReceiveTest.java

```diff
@@ -0,0 +1,219 @@
+package com.microsoft.azure.servicebus.jms.aad;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.jms.Connection;
+import javax.jms.Destination;
+import javax.jms.JMSException;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Queue;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import javax.jms.Topic;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.identity.ClientSecretCredential;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import com.microsoft.azure.servicebus.jms.ServiceBusJmsConnectionFactory;
+
+public class SendReceiveTest 
+{
+	private static final String AUDIENCE = "https://servicebus.azure.net/.default";
+    private static final String AUTHORITY = "https://login.microsoftonline.com/%s/";
```

Comment:
Is there a reason we try not to make the names public and reuse them across files instead of redefining in two places.

Reply:
This is the only place we use them the other ones are separate because that is a test and it was use to simulate user passing token. but they will be gone now that we are no longer supporting it.